package.json

```diff
@@ -10,7 +10,7 @@
     "server:dev": "nodemon index.js",
     "db:build": "node ./db/init_db",
     "start": "node index.js",
-    "test": "jest --runInBand",
+    "test": "jest --runInBand --watchAll --verbose db",
```

Comment:
we probably don't want to hardcode this to call `db` tests only.

Reply:
updated and resolved.